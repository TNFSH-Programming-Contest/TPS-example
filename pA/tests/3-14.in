-388285793 / -675152785
