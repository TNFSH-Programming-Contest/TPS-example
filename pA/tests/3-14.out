-388285793 / -675152785 = 0 ... -388285793
